src/openrct2/config/Config.cpp

```diff
@@ -769,5 +812,13 @@ bool config_find_or_browse_install_directory()
             return false;
         }
     }
+    // While we're at it, also check if the player has RCT1
+    std::string rct1Path = Config::FindRCT1Path();
+    if (!rct1Path.empty())
+    {
+        Memory::Free(gConfigGeneral.rct1_path);
```

Comment:
The Memory class is deprecated. Please use `free` instead.

(I really wish we were at a point where we could leave memory management to C++ intrinsics…)

Reply:
Yes, I didn't get round to removing the remaining ones because there are lots of them.